t/mojo/reactor_ev.t

```diff
@@ -8,307 +8,375 @@ plan skip_all => 'EV 4.32+ required for this test!'                  unless eval
 use IO::Socket::INET;
 use Mojo::Util qw(steady_time);
 
+sub make_reactor {
+  return Mojo::Reactor::EV->new;
+}
+
+sub make_listen {
+  return IO::Socket::INET->new(Listen => 5, LocalAddr => '127.0.0.1');
+}
+
+sub make_server {
+  my $listen = shift or die 'No listen passed.';
```

Comment:
We tend to put the `die` first, and punctuation results in badly formatted error messages.

Reply:
I removed every subroutine I made since they had no sense so fixed.